docs/pages/access-controls/reference.mdx

```diff
@@ -338,41 +338,64 @@ RBAC lets teams limit what resources are available to Teleport users. This can b
 you don't want regular users editing SSO (`auth_connector`) or creating and editing new roles
 (`role`).
 
-List of all rule options defined below.
+Below is an example `allow` section of commonly used `rules`:
 
 ```yaml
-  allow:
-    rules:
+allow:
+  rules:
```

Comment:
Each resource-specific comment includes mostly the same text. Could we make this a bit cleaner by extracting these comments into another paragraph above the sample YAML?

E.g.,

```md
Each rule includes a list of Teleport resources and the CRUD 
operations that a user is allowed to execute on them.
``` 
We could then use comments in the YAML snippet to clarify what each `resources` value means if it's not explicit enough.

Reply:
Yeah good idea, this is unnecessarily repetitive.